packages/forms-web-app/src/lib/views.js

```diff
@@ -95,8 +95,8 @@ const VIEW = {
 	},
 
 	FINAL_COMMENT: {
-		FINAL_COMMENT: 'final-comment/final-comment',
-		COMMENTS_QUESTION: 'final-comment/comments-question'
+		FINAL_COMMENT: 'full-appeal/submit-final-comment/final-comment',
+		COMMENTS_QUESTION: 'full-appeal/submit-final-comment/comments-question'
```

Comment:
Should CHECK_YOUR_ANSWERS be defined here?

Reply:
the view and associated routing for check your answers will be handled in as-5751